src/SeqViewerContainer.tsx

```diff
@@ -125,11 +144,13 @@ class SeqViewerContainer extends React.Component<SeqViewerContainerProps, SeqVie
    */
   getSelection = (
     state: Selection,
-    prop?: {
-      clockwise?: boolean;
-      end: number;
-      start: number;
-    }
+    prop?:
+      | {
+          clockwise?: boolean;
+          end: number;
+          start: number;
+        }
+      | Selection
```

Comment:
this doesn't need to change either, it's 1:1 w/ the prop that users pass in so it doesn't have those other fields.

Looking thru it quickly, a few lines (maybe more) that need to change:
- state field here: https://github.com/Lattice-Automation/seqviz/blob/develop/src/SeqViewerContainer.tsx#L73
- return type here: https://github.com/Lattice-Automation/seqviz/blob/450552df7c53c5bd8dbe9029ffab304a3f8e1537/src/SeqViewerContainer.tsx#L126